packages/tooling/fast-tooling/src/web-components/html-render-layer-navigation/html-render-layer-navigation.spec.ts

```diff
@@ -20,33 +20,79 @@ async function setup() {
     >("fast-tooling-html-render-layer-navigation");
     const message = new MessageSystem({
         webWorker: fastMessageSystemWorker,
-        dataDictionary: dataDictionaryConfig as any,
-        schemaDictionary,
     });
+
+    let hasBeenCalled = false;
+    let intervalInstance;
+
+    async function messageSystemHasBeenCalled() {
```

Comment:
Should there be some sort of a timeout on this to prevent it from possibly hanging?

Reply:
Theres a built in timeout of 5000 for any given test.